referrer-policy/README.md

```diff
@@ -42,7 +43,7 @@ Start from the command line:
 cd ~/web-platform-tests/referrer-policy
 
 # Generate the test resources.
-./generic/tools/generate.py
+./generic/tools/generate.py --target=release
```

Comment:
I'd suggest making `release` the default, and requiring `--target=debug` to build with the more verbose test template.